website/docusaurus.config.js

```diff
@@ -259,7 +259,7 @@ module.exports = {
         },
         blog: {
           showReadingTime: true,
-          editUrl: 'https://github.com/moja-global/community-website/edit/master/website/blog/',
+          editUrl: 'https://github.com/moja-global/community-website/edit/main/website/blog/',
```

Comment:
I don't think this is working, leads to 404, please change this to `https://github.com/moja-global/community-website/blog/main/website/` and the `editUrl` of `blog` to `https://github.com/moja-global/community-website/docs/main/website/`